packages/bridge-ui/src/App.svelte

```diff
@@ -174,20 +174,20 @@
   });
 
   pendingTransactions.subscribe((store) => {
-    store.forEach(async (tx) => {
-      await $signer.provider.waitForTransaction(tx.hash, 1);
+    (async () => {
+      const confirmedPendingTxIndex = await Promise.race(
```

Comment:
This way, we wouldn't be having stale `Promise`s. Earlier stale `Promise`s were resolving and each was triggering a toast which resulted in more than one toast being shown for each pending transaction.